### DISCUSSION TITLE
Management Accounts

### DISCUSSION THREAD

**Discussion by khrome83** [pocketbase/pocketbase#3336] [Q&A] [answered]
As I understand it, there are two types of accounts. 

1. Admins that have access to the UI and can manage the fields, and the data manually.
2. Users who access with the backend systems via JS/Dart SDK integration with a app/site.

I actually want a 3rd. 

A user that can see the admin panel, interact with the data, but can't change schema or setup webhooks, etc. Ideally I want to be able to hide or lock down specific tables, or even specific fields. 

The idea here is that Pocketbase could be used as a CMS or data management tool directly without having to build out additional sites to show the same data. 

Is this possible today? Or does only Admins have access to the Pocketbase interface, and have full control?

**Comment by ganigeorgiev** [comment, marked as answer]
There are no restrictions for the PocketBase admins and they can do everything (including deleting/creating other admins).

What you want is not supported at the moment and you'll have to create your own tailored user admin interface (see also https://github.com/pocketbase/pocketbase/discussions/50).

**Reply by khrome83** [reply]
Really appreciate the fast response. Totally understand that you have to scope Pocketbase.io for 1.0 and beyond. 

I am very impressed with how you designed and built this, and the options available for others to extend. I wish I had the time to pursue that, and may still yet want too. =)
